File: src/main/java/me/miki/shindo/management/screenshot/Screenshot.java
package me.miki.shindo.management.screenshot;

import me.miki.shindo.utils.animation.simple.SimpleAnimation;

import java.io.File;

public class Screenshot {

    private final SimpleAnimation selectAnimation = new SimpleAnimation();

    private final String name;
    private final File image;

    public Screenshot(File image) {
        this.image = image;
        this.name = image.getName().replace(".png", "");
    }

    public SimpleAnimation getSelectAnimation() {
        return selectAnimation;
    }

    public String getName() {
        return name;
    }

    public File getImage() {
        return image;
    }
}

File: src/main/kotlin/me/miki/shindo/management/screenshot/Screenshot.kt
package me.miki.shindo.management.screenshot

import me.miki.shindo.ui.animation.v1.value.SimpleAnimation
import java.io.File

class Screenshot(image: File) {

    private val selectAnimation = SimpleAnimation()
    private val name: String = image.name.replace(".png", "")
    private val image: File = image

    fun getSelectAnimation(): SimpleAnimation = selectAnimation
    fun getName(): String = name
    fun getImage(): File = image
}
